### ISSUE TITLE
Add to Windows installation "Problems you may experience" documentation

### ISSUE THREAD

**Issue by mkaz** [federico-terzi/espanso#203]
I ran into an error installing Espanso that vcruntime140_1.dll was not found.

To fix the issue, install the latest Visual C++ runtime  `vc_redist.x64.exe` from Microsoft:
https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads

Reboot and Espanso worked fine. 👍

**Comment by federico-terzi** [comment]
Hey @mkaz,
Thank you very much for your feedback! Honestly, I'm a bit shocked about this issue as it is surely the result of the last update, which removed the `vc_redist.x64.exe` automatic installation in favor of a "local deployment" of the DLLs (basically it copied the required DLLs in the espanso folder).

Searching through the logs of the CI pipeline, I found the root of the problem. Turns out that the VM didn't include the required `vcruntime_140_1.dll` in the installation. I'll make sure to solve the problem as soon as possible.

Cheers :)
